cluster/external/provider.sh

```diff
@@ -31,6 +31,6 @@ function down() {
 
 function build() {
     # Build code and manifests
-    ${KUBEVIRT_PATH}hack/dockerized "DOCKER_TAG=${DOCKER_TAG} KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER} ./hack/build-manifests.sh"
```

Comment:
Why not just leave everything else and just fix up the external provider here?  So it turns out I can get this to work without the other changes if I fix up the provider script.  It does seem intermittent, but it did complete and I was able to push to my quay.io account.

I'd be happy to s/DOCKER_/CONTAINER_/ it does make logical sense.  I just want to make sure I can get it to work reliably and then update the docs.

Reply:
Or does it not work with dockerhub?  Maybe that's what you're getting at here.

Reply:
I took the pieces I was missing and just converted the external-provider to use the new vars correctly, that worked in terms of getting the images built and uploaded to my registry.  The Bazel process appears to hang though, I get here:

```
Successfully built 728500658bb0
Successfully tagged kubevirt:latest
go version go1.11.5 linux/amd64

go version go1.11.5 linux/amd64
Starting local Bazel server and connecting to it...
INFO: Invocation ID: d483f971-eb50-416c-839c-4c87205c01ca
DEBUG: /root/.cache/bazel/_bazel_root/6f347497f91c9a385dcd9294645b76e0/external/bazel_skylib/skylark_library.bzl:23:1: WARNING: skylark_library.bzl is deprecated and will go away in the future, please use bzl_library.bzl instead.
INFO: Analysed target //:push-images (543 packages loaded, 13139 targets configured).
INFO: Found 1 target...
Target //:push-images up-to-date:
  bazel-bin/push-images
INFO: Elapsed time: 80.004s, Critical Path: 35.15s
INFO: 45 processes: 45 processwrapper-sandbox.
INFO: Build completed successfully, 63 total actions
INFO: Build completed successfully, 63 total actions
```
Your full PR here results int he same behavior.  It doesn't appear that anything is attempting to deploy to the K8s cluster in this case.